services/apps/data_sink_worker/src/service/member.service.ts

```diff
@@ -232,6 +232,16 @@ export default class MemberService extends LoggerBase {
         if (dbMember) {
           this.log.trace({ memberId: dbMember.id }, 'Found existing member.')
 
+          // set a record in membersSyncRemote to save the sourceId
+          // we can't use member.attributes because of segments
+          if (member.attributes?.sourceId?.[platform]) {
```

Comment:
member identities can now also have sourceId if it's needed

Reply:
I updated processData so that sourceId is also inserted into the identity - However, I didn't change this logic because we also have this for organizations (and we don't have identities for organizations yet). I think it'll be less confusing if we get the sourceIds from syncRemote tables for both entities when pushing data